src/main/scala/exu/execution-units/functional-unit.scala

```diff
@@ -605,6 +605,7 @@ class IntToFPUnit(latency: Int)(implicit p: Parameters)
   req.in2 := unbox(io_req.rs2_data, tag, None)
   req.in3 := DontCare
   req.typ := ImmGenTyp(io_req.uop.imm_packed)
+  req.fmt := DontCare // FIXME: this may not be the right thing to do here
```

Comment:
Without assigning `req.fmt` here, I hit:
```
[error] (run-main-0) firrtl.passes.PassExceptions: 
[error] firrtl.passes.CheckInitialization$RefNotInitializedException:  @[functional-unit.scala 598:17] : [module IntToFPUnit]  Reference req is not fully initialized.
[error]    : req.fmt <= VOID
```
in https://app.circleci.com/pipelines/github/ucb-bar/chipyard/3315/workflows/c3d06b70-4b7d-4229-bdba-3e86c79a2e25/jobs/44012

Assigning them to `DontCare` gets past that exception but then test `rv64ud-p-fadd` fails (possibly along with others) in `make -C sims/firesim/sim TARGET_CONFIG=DDR3FRFCFSLLC4MB_FireSimLargeBoomConfig run-asm-tests`

Reply:
@jerryz123 the tests (`make -C sims/firesim/sim TARGET_CONFIG=DDR3FRFCFSLLC4MB_FireSimLargeBoomConfig run-asm-tests`) pass with `req.fmt := DontCare` here.  Should I remove the FIXME comment?
```suggestion
  req.fmt := DontCare
```

or should we assign `req.fmt` the same way you're assigning it in fpu.scala in e1a70af ?